NashTeamAlpha/BangazonWeb
# Gr adding products

## Pull Request Description

1. Added an ajax call that handles the AddToCart Method and redirects the user to the index.
1. Changed the AddToCart method in the Products Controller to return a Json object.

## Steps to Test

1. git fetch.
1. git checkout gr-AddingProducts
1. Make sure that you can add products to an order on your local machine.
